resources_pages/quiz.md

```diff
@@ -5,81 +5,95 @@ title: MDS quizzes
 - You will see a list of future quizzes, including quiz times and locations, under Exam reservations.
 
-### Format
+### Quiz Logistics
+
+- You must register for a specific quiz time slot using PrairieTest. This will typically occur the week before quizzes.
+- All MDS quizzes are closed-book and closed-internet, with the exception of a digital cheatsheet.
+- Communication with anyone or any external resources during the quiz is strictly prohibited. For details, please visit our [Quiz policies](https://ubc-mds.github.io/policies/#quiz-policies) and [Plagiarism](https://ubc-mds.github.io/policies/#plagiarism) pages.
+- You must attend your assigned quiz time and location. Failure to check in at the correct time and location may result in missing the quiz.
+- When instructed to leave the room after the quiz, please do so quietly to avoid disrupting others. Refrain from discussing the - quiz with anyone until everyone has completed it.
 
-Quizzes will typically be through [Canvas](canvas.ubc.ca). Unlike labs, quizzes will be performed within a defined time period and will be supervised by MDS instructional staff.
+### Quiz Room Procedures
 
 #### Upon arriving at the quiz room
 
-1. You must attend the assigned quiz session in your designed quiz room (check [PrairieTest](https://ca.prairietest.com/pt) if you are unsure).
-2. You must be present at least **10 minutes** before the quiz starts.
-3. Upon entering the exam room, you should **not** communicate with anyone other than the invigilators.
-4. An invigilator will check you in. Please wait until the reader displays **red** light before tapping your student card.
-5. You must put all of your personal belongings and electronics on the racks at the front of the room—phones, bags, hats, jackets, smartwatches, etc. Phones must be turned off or on airplane mode.
-  - Exceptions: You are allowed to bring a double sided A4 cheatsheet and we will provide scrap papers.
-  - Exceptions: You can bring a scientific calculator if you wish, but there is an embedded calculator as well.
-6. Find a seat and log onto the provided computer using the username and password on the whiteboard or on the wall.
+1. Attend the assigned quiz session in your designated quiz room (check [PrairieTest](https://ca.prairietest.com/pt) if you are unsure).
+2. Be present at least **10 minutes** before the quiz starts.
+3. Upon entering the exam room, do **not** communicate with anyone other than the invigilators.
+4. An invigilator will check you in. Wait until the reader displays a **red** light before tapping your student card.
+5. Place all personal belongings and electronics on the racks at the front of the room (phones, bags, hats, jackets, smartwatches, etc.). Phones must be turned off or on airplane mode.
+  - Exceptions: You are allowed to upload a digital double-sided A4 cheatsheet. Scrap papers will be provided by the exam room.
```

Comment:
Maybe change to:
- Note: Your 2-page A4 cheatsheet must be digital, and scrap papers will be provided in the exam room.